irispy/sji.py

```diff
@@ -502,16 +515,16 @@ def annotate_frame(i):
             axes.set_title("{s.name}".format(s=self[i]))
 
             # x-axis label
-            if self[0].coordinate_system.x == 'HG':
-                xlabel = 'Longitude [{lon}'.format(lon=self[i].spatial_units.x)
+            if self[0].coordinate_system.axis1 == 'HG':
```

Comment:
This should be re-written to use this: https://github.com/sunpy/sunpy/blob/master/sunpy/visualization/visualization.py#L38 helper function from SunPy. As map now uses here: https://github.com/sunpy/sunpy/blob/master/sunpy/map/mapbase.py#L1720

Reply:
fixed